TUnit.Engine/Capabilities/TrxReportCapability.cs

```diff
@@ -5,8 +5,9 @@ namespace TUnit.Engine.Capabilities;
 internal class TrxReportCapability : ITrxReportCapability
 {
     public void Enable()
-    {
-    }
+        => IsTrxEnabled = true;
```

Comment:
Does this get called by the non-abstractions Trx package?

Reply:
No. It's not part of `ITrxReportCapability` interface. It's read in places that create the Trx-related messages so that they can be skipped if trx isn't enabled.

Reply:
I mean what actually calls enable

Reply:
Ah, `Enable` itself is called by the Trx extension (the non-abstractions package):

https://github.com/microsoft/testfx/blob/26b1f1b86bbaa1c524e4b91c6bbd838687ea23cc/src/Platform/Microsoft.Testing.Extensions.TrxReport/TrxDataConsumer.cs#L206

https://github.com/microsoft/testfx/blob/26b1f1b86bbaa1c524e4b91c6bbd838687ea23cc/src/Platform/Microsoft.Testing.Extensions.TrxReport/TrxDataConsumer.cs#L80